prime-router/docs/proposals/0017-metadata-tables-cleanup/elr-metadata-table-changes.sql

```diff
@@ -0,0 +1,381 @@
+/**
```

Comment:
Oops, I didn't realize the SQL was all repeated in the proposal and in a .sql doc.